src/build/emitter.ts

```diff
@@ -672,7 +672,10 @@ export function emitWebIdl(
   function paramsToString(ps: Browser.Param[]) {
     function paramToString(p: Browser.Param) {
       p = resolvePromise(p);
-      if (p.name.toLowerCase().includes("url") && p.type === "USVString") {
+      if (
+        (p.name.toLowerCase().includes("url") && p.type === "USVString") ||
+        p.type === "RequestInfo"
```

Comment:
This is now a little bit long to be in a if clause, could you add a function for this and call it here?

Reply:
Sure.